README.md

```diff
@@ -1 +1,1148 @@
-# balena-cli-masterclass
\ No newline at end of file
+balenaCLI Masterclass
+=====================
+
+**Masterclass Type:** Core
+
+# Introduction
+
+The balena Command Line Interface (balenaCLI) utility consists of a number
+of commands that allow a user to develop, deploy and manage balena applications
+and devices.
+
+Everything that can be achieved via the balenaCloud dashboard can also be
+achieved via the balenaCLI.
+
+In this masterclass, you will learn how to:
+
+* Login to your account
+* Push application code to a balena Application
+* Deploy locally built code to a balena Application
+* SSH into a balena device
+* Push and build applications on a device on the local network for fast
+    development and prototyping
+* Use private Docker registries and modules from a balena device
+
+# Hardware and Software Requirements
+
+It is assumed that the reader has access to the following:
+
+* A [balenaFin 1.1](https://store.balena.io/collections/developer-kit/products/balenafin-v1-1-0-developer-kit)
```

Comment:
Is there any reason to specifically require the fin here? Presumably we could use the fin as an example (and should, for subliminal messaging!) but we're making this masterclass inaccessible for all those who may not already have a fin, when to me it seems like a RPi3/NUC might do just fine (given the lack of hw requirements).

Reply:
I'd really only put that in because we were going to give every engineer a Fin. I think you're right though, if we just say a suitable device. It does make a little difference to the emulated build stuff, I'll have a think and rewrite that section appropriately.

Reply:
Done.